src/main/java/org/craftercms/studio/controller/rest/v2/ConfigurationController.java

```diff
@@ -111,7 +111,7 @@ public ResponseBody getConfiguration(@ValidSiteId @RequestParam(name = "siteId",
 
     @Valid
     @PostMapping("/write_configuration")
-    public ResponseBody writeConfiguration(@Valid @RequestBody WriteConfigurationRequest wcRequest)
+    public ResponseBody writeConfiguration(@Validated @RequestBody WriteConfigurationRequest wcRequest)
```

Comment:
Notice that @Validated is meant to work as an extension of @Valid. For some reason, custom validator is being triggered twice per request when using @Validated

Reply:
Do you mean @valid causes double invocation, so you used @validated?

Reply:
Yes